File: Portfolio.Data/Entities/Folder.cs
using System.Collections.Generic;

namespace Portfolio.Data.Entities
{
    public class Folder
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public bool IsDeleted { get; set; }

        public User User { get; set; }

        public List<FolderUpload> FolderUploads { get; set; }
    }
}

File: Portfolio.Web/ClientApp/src/app/models/api/folder.ts
import { FolderUpload } from './folder-upload';
import { User } from './user';

export interface Folder {
  id: number;
  userId: number;
  name: string;
  description: string;
  isDeleted: boolean;

  user: User;

  folderUploads: FolderUpload[];
}
